src/member/member.ts

```diff
@@ -51,6 +51,8 @@ export type Member = {
 export type CompleteMember = Member & {
   type: `${MemberType}` | MemberType;
   extra: MemberExtra;
+  enableSaveActions?: boolean;
+  userAgreements?: string;
```

Comment:
Should this be `userAgreementDate`

Reply:
It could be but I have to change in the backend too.